app/jobs/print_to_paper_job.rb

```diff
@@ -2,14 +2,11 @@ class PrintToPaperJob < ApplicationJob
   queue_as :default
 
   def perform(text)
-    # puts("This is a placeholder for the printing to paper job. The text to be printed would be this:")
-    # puts(text)
-
-    # file = Tempfile.new
-    # file.write text
-    # file.rewind
-
-    # This is just a placeholder for a possible call to the printer:
-    # system("lpr -P myprintername #{file.path}")
+    Net::SSH.start(Rails.application.credentials.printing_raspi["hostname"], Rails.application.credentials.printing_raspi["username"], password: Rails.application.credentials.printing_raspi["password"]) do |ssh|
+      # Print command here
+      pwd = ssh.exec!("pwd")
```

Comment:
@raphael-arce, if we follow the 2 Raspi approach, we could SSH into the printing one and execute whichever command we wanna use for printing here (`pwd` is just an example here). Tested this by running this method on my local machine and it successfully tunnelled into my Raspi here!

Reply:
amazing 👍  i'm experimenting with printing commands right now!